charts/datadog/README.md.gotmpl

```diff
@@ -53,13 +57,13 @@ helm install --name <RELEASE_NAME> \
     datadog/datadog
 ```
 
-#### Create and provide a secret that contains your Datadog API Key
+#### Create and provide a secret that contains your Datadog API and APP Keys
 
 To create a secret that contains your Datadog API key, replace the <DATADOG_API_KEY> below with the API key for your organization. This secret is used in the manifest to deploy the Datadog Agent.
 
 ```bash
-DATADOG_SECRET_NAME=datadog-secret
-kubectl create secret generic $DATADOG_SECRET_NAME --from-literal api-key="<DATADOG_API_KEY>" --namespace="default"
+DATADOG_API_SECRET_NAME=datadog-api-secret
+kubectl create secret generic $DATADOG_API_SECRET_NAME --from-literal api-key="<DATADOG_API_KEY>" --namespace="default"
```

Comment:
I’m wondering if we should keep the `--namespace="default"`.

It gives the feeling that, even if the `datadog-agent` is running in its own namespace, the secret would need to be in `default`.

```suggestion
kubectl create secret generic $DATADOG_API_SECRET_NAME --from-literal api-key="<DATADOG_API_KEY>"
```